packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6907,84 +6893,69 @@ export default class Meeting extends StatelessWebexPlugin {
   }
 
   /**
-   * Publishes a track.
+   * Publishes a stream.
    *
-   * @param {LocalTrack} track to publish
+   * @param {LocalStream} stream to publish
+   * @param {MediaType} mediaType of the stream
    * @returns {Promise}
    */
-  private async publishTrack(track?: LocalTrack) {
-    if (!track) {
```

Comment:
there are many possible cases where publishStream() will be called with stream being undefined, that's why the old code had that `if (!track)` check, same check for stream is needed in the new code (and type in function arguments also needs updating)

Reply:
Done: https://github.com/webex/webex-js-sdk/pull/3052